src/pyodide/emscripten_setup.capnp

```diff
@@ -0,0 +1,13 @@
+@0xc00ad00cc650fb45;
+
+struct EmscriptenSetup {
+  code @0 :Text;
+  pyodideAsmWasm @1 :Data;
+  pythonStdlibZip @2 :Data;
+}
+
+const emscriptenSetup :EmscriptenSetup = (
+  code = embed "emscriptenSetup.js",
+  pyodideAsmWasm = embed "pyodide.asm.wasm",
+  pythonStdlibZip = embed "python_stdlib.zip",
+);
```

Comment:
It would be nice to put this and `pyodide.capnp` together into the same capnproto.bin file so we don't have to upload/download two separate things.

Reply:
The benefit of uploading two separate things is they are cached separately.
If we suspect this to change less often than the latter we'll get a big performance improvement.
This is a huge package (>10MB) while the other package is tiny.

Reply:
But then in order to get the performance improvement we'll need to make extra tooling to keep track of when the big package hasn't actually changed. If we try to do it manually we'll probably mess up.

Reply:
Not sure how that's different tooling to what we have now for the regular package

Reply:
But I suppose if we upload separately, we can get that tooling working later.

Reply:
What I mean is either we always update both at the same time and then there is no caching benefit to separating them, or we need to somehow know when the bigger bundle hasn't changed.